docs/content/ingestion/firehose.md

```diff
@@ -147,3 +147,34 @@ An example is shown below:
 |type|This should be "timed"|yes|
 |shutoffTime|time at which the firehose should shut down, in ISO8601 format|yes|
 |delegate|firehose to use|yes|
+
+#### SqlFirehose
+
+SqlFirehoseFactory can be used to ingest events residing in RDBMS. The database connection information is provided as part of the ingestion spec.
+An example is shown below:
+
+```json
+{
+    "type" : "sql",
+    "database": {
+        "type": "mysql",
+        "connectorConfig" : {
+        "connectURI" : "jdbc:mysql://host:port/schema",
+        "user" : "user",
+        "password" : "password"
+        }
+     },
+    "sqls" : ["SELECT * FROM table1", "SELECT * FROM table2"]
```

Comment:
Please add a description about what this means and how it works.

Reply:
Added description.